src/main.js

```diff
@@ -35,23 +34,15 @@ let load = loader({
 
   monitor: {
     requires: ['cfg', 'drain'],
-    setup: ({cfg, drain}) => base.stats.startProcessUsageReporting({
-      drain:      drain,
-      component:  cfg.app.statsComponent,
-      process:    'server'
+    setup: ({cfg, drain}) => base.monitor({
+      project:      'taskcluster-auth',
+      drain:        drain,
+      credentials:  cfg.taskcluster.credentials,
+      process:      'server',
```

Comment:
I don't think this should be set to `'server'` manually even if it was before. The `loader` can take `process` in `requires` and in `setup` the same way as `cfg` and `drain`. Then you can just pass it in here!